src/components/incident/IncidentDetails.tsx

```diff
@@ -106,6 +106,14 @@ const isValidUrl = (url: string) => {
   return true;
 };
 
+const formatUrlOrIdentity = (url: string, title: string, empty: string = "") => {
+  if (isValidUrl(url || "")) {
+    return <a href={url}>{title}</a>;
+  } else {
+    return url || empty;
```

Comment:
Instead of returning `url || empty` here, you could just do 
`details={url && <a href={incident.details_url}>More details</a>}` if you want empty string on empty url string or 
`details={url && <a href={incident.details_url}>More details</a> || "-"}` if you want "-" when url is "" (empty string).
The function name "formatUrlorIdentity" is misleading, instead `hyperlinkIfUrl(url, title)` or something and instead of returning `url || empty` just use it like this `hyperlinkIfUrl(url, title) || "-"`.

Reply:
Ugh, copypaste-bug `isValidUrl(url || "")` should be `isValidUrl(url)`. We only want a link if the url is absolute.

Reply:
I tried doing stuff right in the return but got so many errors it was easier to make a function.

Reply:
`isValidUrl(url) && <a href={incident.details_url}>More details</a> || "-"` maybe.

Reply:
I went with renaming and simplifying the function.